distributor-api-v1/operations.md

```diff
@@ -404,7 +404,7 @@ TBC
 
 A localized text is an object of the property values localized into languages supported by hotel, indexed by language codes.
 
-#### Payment Gateway   <a id="payment-gateway"></a>
+#### PaymentGateway   <a id="payment-gateway"></a>
```

Comment:
But now it's still inconsistent, for example Product charging, Product posting. All these should be camelCase one word. What do you think? @dobras  @dmarchuk

Reply:
@dobras @zdrazil Most of the things in this document are one word PascalCase, so I think we should make it all consistent in that way. Also in the API everything is PascalCase, so I think it makes sense to keep it that way.

Reply:
Sorry, yeah I've meant PascalCase. But Product charging, Product posting. Those are not written in PascalCase. So it should be unified. What do you think?

Reply:
Added to the followup issue